doc/user_manual/include/Knapsack.tex

```diff
@@ -9,12 +9,13 @@ \section{Knapsack Models}
 find the optimal solution.
 
 
-\subsection{BaseKnapsackModel}
+\subsection{BaseKnapsack Model}
```

Comment:
BaseKnapsackModel --> SimpleKnapsackModel? If so, please update this section.

Reply:
good catch, fixed all section